scripts/context.js

```diff
@@ -94,6 +96,10 @@ function singlePageView() {
           }
         }
       }
+      //Show error message if no context is selected
+      if (countFeature <= 0) {
+        $('#error-message').removeClass('display-none')
```

Comment:
You can use show() and hide() method of jquery

Reply:
I mean add display none by default and you can use `hide` or `show` jquery methods instead of `addClass` and `removeClass`. I know using both the methods will result the same thing that was happening before. Thank you